### PR TITLE
Arabic translation + support RTL layout

### PR DESCRIPTION
**Pull request by karmac2015**
Translate the most strings to arabic, and change layouts to support RTL

### CODE REVIEW

File: app/src/main/res/values-ar/strings.xml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    <string name="manage_blocked_numbers">إدارة الأرقام المحظورة</string>
+    <string name="not_blocking_anyone">لم تحظر أي شخص.</string>
+    <string name="add_a_blocked_number">إضافة رقم محظور</string>
+    <string name="block_number">حظر رقم</string>
+    <string name="block_numbers">حظر أرقام</string>
+    <string name="blocked_numbers">أرقام محظورة</string>
+    <string name="instant_messaging">المراسلة الفورية (IM)</string>
+    <string name="on_contact_click">عند النقر فوق جهة اتصال</string>
+    <string name="release_10">السماح بتخزين جهات الاتصال في قاعدة بيانات محلية ، مخفية عن التطبيقات الأخرى</string>
+</resources>
```
Comment (tibbi) [inline app/src/main/res/values-ar/strings.xml]: keep the same string structure as in the original file, do not group them together. Also, why have you deleted the file below? keystore.properties_sample

Comment (karmac2015) [reply app/src/main/res/values-ar/strings.xml]: I used translation editor in android studio to manage strings, if you want it grouped together I will try to grouping them. I rename keystore.properties_sample to keystore.properties in my local project to run it on my device.

Comment (tibbi) [reply app/src/main/res/values-ar/strings.xml]: keep the same strings structure and revert the keystore changes

**Comment by karmac2015** [comment]
I rewrite the strings like you want, and I add the file keystore.properties_samples again

**Comment by tibbi** [comment]
seems good, I will maybe do a few more tweaks. Thanks

**Pull request merged**